aerospike/content.md

```diff
@@ -2,174 +2,315 @@
 
 ## What is Aerospike?
 
-[Aerospike](http://aerospike.com) is a distributed NoSQL database purposefully designed for high performance web scale applications. Aerospike supports key-value and document data models, and has multiple data types including List, Map, HyperLogLog, GeoJSON, and Blob. Aerospike's patented hybrid memory architecture delivers predictable high performance at scale and high data density per node.
+[Aerospike](http://aerospike.com) is a distributed NoSQL database purposefully designed for high performance applications. Aerospike supports key-value and document data models, and has multiple data types including List, Map, HyperLogLog, GeoJSON, and Blob. Aerospike's patented hybrid memory architecture delivers predictable high performance at scale and high data density per node.
 
 %%LOGO%%
 
+-	[Getting Started](#getting-started)
+	-	[Running an Aerospike Server node](#running-an-aerospike-server-node)
+	-	[Enterprise Edition](#enterprise-edition)
+	-	[Community Edition](#community-edition)
+-	[Connecting to your Aerospike Container](#connecting-to-your-aerospike-container)
+	-	[Using aql](#using-aql)
+	-	[Using asadm](#using-asadm)
+-	[Customizing the Default Developer Environment](#customizing-the-default-developer-template)
+	-	[List of template variables](#list-of-template-variables)
+	-	[Preconfigured namespace](#preconfigured-namespace)
+-	[Advanced Configuration](#advanced-configuration)
+	-	[Persistent data directory](#persistent-data-directory)
+	-	[Block storage](#block-storage)
+	-	[Persistent Lua cache](#persistent-lua-cache)
+	-	[A note about security](#a-note-about-security)
+	-	[Networking](#networking)
+	-	[Configuring the node's access address](#configuring-the-nodes-access-address)
+-	[Sending Performance Data to Aerospike](#sending-performance-data-to-aerospike)
+	-	[Mesh clustering](#mesh-clustering)
+-	[Image Versions](#image-versions)
+	-	[ee-[version]](#ee-version)
+	-	[ce-[version]](#ce-version)
+-	[Reporting Issues](#reporting-issues)
+-	[License](#license)
```

Comment:
This probably isn't going to work correctly in Docker Hub's rendering. :grimacing:

Reply:
I copied that from markdonwfmt's diffs. I agree, it looks hideous, I hope I was wrong.

Reply:
Oh, such links work on github - what would be the method here?

Reply:
I like the leading tab, just not a fan of the tab after the bullet. I try to avoid tabs after fixed space characters. In this case it is "ok" since we know there is always exactly one fixed space character, but seems like an unnecessary exception to make.

Somewhat relevant:
[tabs vs spaces vs both](https://www.emacswiki.org/pics/static/TabsSpacesBoth.png)

Reply:
This file exists to be the source-of-truth for Docker Hub, so I'd suggest dropping the whole TOC - if this file is long enough to need a TOC, it should probably be split up and have most of it live elsewhere (and have this file link to it).

Reply:
Especially since we have a 25k character length limit on Docker Hub descriptions after which they'll be cut off anyways.

Reply:
Alright - done.

Reply:
:sweat: ... 10K characters to spare.